package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "cbioportal-frontend",
   "private": true,
+  "version": "3.2.13-alpha.0",
```

Comment:
added version back to make `updatePackageVersion` also update the root `package.json` dependencies. If the version is missing, lerna ignores this file completely when updating other package versions.

Reply:
when do we need to update this? In every PR?

Reply:
It is not critical to update this version every time. The main reason to have a version in this `package.json` is to make `lerna version` to properly update dependencies. It would be nice to keep this version in sync with the tagged master version though.